### PR TITLE
Add/View Comments

### PR DESCRIPTION
**Pull request by antoniolinhart**
- Add functionality to add comments
- Store comments in datastore
- View comments in text bubble-like form

Image of feature:
comments-pic1

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+/** Servlet responsible for creating new comments. */
+@WebServlet("/new-comment")
+public class NewCommentServlet extends HttpServlet {
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String author = request.getParameter("author-input").trim();
+    // replace author name if empty
+    if(author.length() == 0) author = "Anonymous User";
```
Comment (VivekShankar) [inline portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java:34]: nit: Add a space between `if` and `author.length()`

File: portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for creating new comments. */
+@WebServlet("/new-comment")
+public class NewCommentServlet extends HttpServlet {
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String author = request.getParameter("author-input").trim();
```
Comment (VivekShankar) [inline portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java:32]: Good use of trim to remove extraneous whitespace!

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -120,6 +121,36 @@ img {
...
+.comment-list {
+  display: block;
+  list-style-type: none;
+  margin-left: 25%;
+  margin-right: 25%;
+  padding-left: 0px;
+  width: auto;
+}
+
+.comment-list > li > p {
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/style.css:137]: Great use of child combinators in CSS to apply different styles to the paragraph vs. span elements inside your comment list! I didn't know about child combinators until now; thanks for showing me a cool feature :)!

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -120,6 +121,36 @@ img {
...
+  background: #0275d8;
+  border-radius: 25px;
+  -moz-border-radius: 25px;
+  -webkit-border-radius: 25px;
+  color: white;
+  flex-wrap: wrap;
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/style.css:143]: Good use of flex-wrap to wrap long comments!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (antoniolinhart) [reply portfolio/src/main/webapp/style.css:137]: Thanks! I didn't know about them either before this week

**Pull request merged**